cli/src/commands/extensions/mod.rs

```diff
@@ -73,8 +73,14 @@ pub async fn handle_extensions(matches: &ArgMatches) -> CommandResult {
 
 /// Handle the `extension add` subcommand path.
 /// Add the extension from the specified path.
-async fn handle_add_extension(path: PathBuf) -> CommandResult {
-    let extension = Extension::try_from(path)?;
+async fn handle_add_extension(path: &str) -> CommandResult {
+    // NOTE: Extension installation without slashes is reserved for the marketplace.
+    if !path.contains('/') {
```

Comment:
#356 includes an example for Windows:
```
phylum extension add C:\dev\foo   # Install from local directory `C:\dev\foo`
```
Will this work for Windows' backslash path separator? I'm still learning Rust, but my guess is that since the type of `path` is a `&str`, it means a literal backslash. Python has the notion of [`os.sep`](https://devdocs.io/python~3.7/library/os#os.sep) to represent
> The character used by the operating system to separate pathname components. This is '/' for POSIX and '\\' for Windows.

Maybe there is something similar for Rust?

Reply:
Path types in rust support both `/` and `\` on Windows, so it's not enough to use `MAIN_SEPARATOR`.

Officially, I think what you're looking for is [`is_separator`](https://doc.rust-lang.org/std/path/fn.is_separator.html) which returns `true` for both `/` and `\` on Windows.

Practically, it's probably fine to just check for both `/` and `\`.

Reply:
```suggestion
    if !path.contains('/') && !path.contains('\\') {
```